package.json

```diff
@@ -35,12 +35,12 @@
     "ember-cli-mocha": "0.13.2",
     "ember-cli-shims": "^1.0.2",
     "ember-cli-uglify": "^1.2.0",
-    "ember-code-snippet": "^1.8.0",
+    "ember-code-snippet": "^1.10.0",
```

Comment:
I'm assuming this is being changed because that is the version that is being installed when `npm install` is ran, and that all of the other dependencies and dev dependencies listed are already at their respective installed version values?